jekyll/_cci2/configuration-reference.md

```diff
@@ -244,7 +249,12 @@ Each `run` declaration represents a new shell. It's possible to specify a multi-
       make test
 ```
 
-Note that our default `shell` has the `-e` option, which causes commands to:
+##### **Shell options**
+**Note** that our default `shell` has a few options enabled by default.
```

Comment:
This's pretty important section, but I guess we would be fine without this "Note that"